tests/test_doxygen.py

```diff
@@ -62,3 +62,12 @@ def test_git_warning(self):
         self.assertRegexpMatches(fake_out.getvalue(), duterr1)
         self.assertRegexpMatches(fake_out.getvalue(), duterr2)
 
+    def test_sphinx_deprecation_warning(self):
+        duterr1 = "testfile.c:6: warning: group test: ignoring title \"Some test functions\" that does not match old title \"Some freaky test functions\"\n"
+        dut = "/usr/local/lib/python3.5/dist-packages/sphinx/application.py:402: RemovedInSphinx20Warning: app.info() "\
+            "is now deprecated. Use sphinx.util.logging instead. RemovedInSphinx20Warning)\n"
+        dut += duterr1
+        with patch('sys.stdout', new=StringIO()) as fake_out:
+            self.warnings.check(dut)
+        self.assertEqual(self.warnings.return_count(), 1)
```

Comment:
this here should be 0 as you are testing doxygen checker and none fo these is doxygen warning

Reply:
`duterr1` contains a doxygen warning. Do you want this variable removed from the test case?

Reply:
no, you are correct, I missed this.